Makefile

```diff
@@ -0,0 +1,11 @@
+
+all: grpc
+
+grpc:
+	python -m grpc_tools.protoc --proto_path=`pwd`/quarkchain/cluster --python_out=`pwd`/quarkchain/cluster --grpc_python_out=`pwd`/quarkchain/cluster  grpc.proto
```

Comment:
how about putting all the generated files into `quarkchain/generated`? also the path to `grpc.proto` seems wrong.

Reply:
done